spec/controllers/animals_controller_spec.rb

```diff
@@ -75,6 +103,35 @@
           it 'la cantidad de animales se mantiene igual' do
             expect { post :create, animal: params, format: 'json' }.not_to change { Animal.count }
           end
+
+          it 'devuelve un error' do
+            post :create, animal: params, format: 'json'
+            expect(parse_response(response)['error']['chip_num'][0]).to eq('ya está en uso')
+          end
+        end
+
+        context 'con nombre en blanco' do
+
+          before(:each) do
+            request.headers['X-USER-TOKEN'] = user.authentication_token
+          end
+        
+          let(:name)  { '' }
+          let(:chip_num) { '' }
+
+          it 'devuelve un estado 422' do
+            post :create, animal: params, format: 'json'
```

Comment:
Acá también: `post :create, animal: params, format: 'json'` aparece en los 3 it del context